sdk/eventhub/azure-eventhub/azure/eventhub/aio/_eventprocessor/event_processor.py

```diff
@@ -121,9 +121,9 @@ def __repr__(self) -> str:
                 task.cancel()
```

Comment:
Add a log here to tell which ones are really cancelled?

Reply:
new log added